src/Microsoft.Data.SqlClient/ref/Microsoft.Data.SqlClient.Batch.cs

```diff
@@ -0,0 +1,127 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Data.SqlClient
+{
+    /// <include file='../../../doc/snippets/Microsoft.Data.SqlClient/SqlBatch.xml' path='docs/members[@name="SqlBatch"]/SqlBatch/*'/>
+    public class SqlBatch : System.Data.Common.DbBatch
```

Comment:
Since we are only implementing the new batch APIs in netcore, we don't want these (through line 120) in the ref file, right?

Reply:
if we ever wanted to implement it on netfx we'd need it. removed.